src/app/home.js

```diff
@@ -58,5 +65,6 @@ export default (menu) => r(
       ['section', {className: 'cards'},
         ...menu.map((item, index) => [MenuItem, {key: index, item}]),
       ],
+      ['section', {className: 'text_with_link'}, [Text, {textWithLink}]],
```

Comment:
here you can destructure the properties into two, so that the component gets them nice and clean